dev-client/app.config.ts

```diff
@@ -101,7 +104,7 @@ export default ({config}: ConfigContext): ExpoConfig => ({
   ios: {
     bundleIdentifier: 'org.terraso.test.Terraso-LandPKS',
     buildNumber: '0',
-    icon: 'src/assets/landpks-square-1024.png',
```

Comment:
question: Why are we no longer using a square image? Aren't iOS icons supposed to be square with rounded edges as opposed to circular?

Reply:
This is per @CourtneyLee333's request and was approved/decided on.